anta/tests/routing/bgp.py

```diff
@@ -447,3 +447,53 @@ def test(self) -> None:
 
         if failures:
             self.result.is_failure(f"Failures: {list(failures.values())}")
+
+
+class VerifyBGPAuthentication(AntaTest):
```

Comment:
Since we only support MD5 in this test, I would call it something like `VerifyBGPMD5Auth` or something and adjust the docstring accordingly.

Reply:
Or maybe `VerifyBGPNeighborMD5Auth`